recipes/libgettext/all/conanfile.py

```diff
@@ -1,29 +1,36 @@
 from conans import ConanFile, AutoToolsBuildEnvironment, VisualStudioBuildEnvironment, tools
 import os
 
-required_conan_version = ">=1.33.0"
+required_conan_version = ">=1.43.0"
 
 
 class GetTextConan(ConanFile):
     name = "libgettext"
     description = "An internationalization and localization system for multilingual programs"
-    topics = ("conan", "gettext", "intl", "libintl", "i18n")
+    topics = ("gettext", "intl", "libintl", "i18n")
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://www.gnu.org/software/gettext"
     license = "GPL-3.0-or-later"
-    deprecated = "gettext"
```

Comment:
any motivation why it's no longer deprecated if we have `gettext` package?

Reply:
Because `gettext` recipe doesn't build & package the lib.